app/src/main/res/values/strings.xml

```diff
@@ -23,13 +23,13 @@
     <string name="share_state_transfer_complete">Transfer complete</string>
     <string name="share_state_stopping">Stopping…</string>
     <string name="share_state_error">Error</string>
-    <string name="share_state_error_text">An unexpected problem has occurred.</string>
+    <string name="share_state_error_text">Unexpected problem.</string>
     <string name="share_button_start">Start Sharing</string>
-    <string name="share_button_starting">Starting… tap to cancel</string>
+    <string name="share_button_starting">Starting… Tap to cancel.</string>
     <string name="share_button_stop">Stop Sharing</string>
     <string name="share_button_complete">Start Sharing Again</string>
-    <string name="share_button_stopping">Stopping… wait until stopped</string>
-    <string name="share_button_error">Try again</string>
+    <string name="share_button_stopping">Stopping… (Wait till it has stopped.)</string>
+    <string name="share_button_error">Retry</string>
```

Comment:
There is a "Retry" below too…

Reply:
Considering the description for it is mentions error, I would expect this to be used in the context of like a title?